File: frontend/lib/categories.ts
// App-side seed list (PRD §8) — mirrors backend/src/categories.py. Kept in sync
// by hand; both change only via deploy. Not a database table at MVP.
export const CATEGORIES = [
  'HVAC',
  'Electrician',
  'Plumber',
  'Lawn/Landscaping',
  'Dentist',
  'General Contractor',
  'Roofing',
  'Pest Control',
  'House Cleaning',
  'Auto Repair',
  'Painter',
  'Handyman',
] as const;

File: backend/src/categories.py
# App-side seed list (PRD §8) — NOT a database table. Adding/renaming a category
# is a code deploy at MVP. The frontend keeps its own copy for the dropdown.
CATEGORIES = [
    "HVAC",
    "Electrician",
    "Plumber",
    "Lawn/Landscaping",
    "Dentist",
    "General Contractor",
    "Roofing",
    "Pest Control",
    "House Cleaning",
    "Auto Repair",
    "Painter",
    "Handyman",
]

CATEGORY_SET = frozenset(CATEGORIES)
